audit_exceptions/versioned_keg_only_allowlist.json

```diff
@@ -19,5 +19,6 @@
   "numpy@1.16",
   "openssl@1.1",
   "pangomm@2.46",
+  "pyqt@5",
```

Comment:
Can I install both `pyqt` and `pyqt@5` at the same time without any linking issues?

Reply:
`pyqt` and `pyqt@5` use different namespace(`pyqt5` and `pyqt6`) and the tools also has a major version suffix.